data/isoppfolgingsplan/foresporsel-oppfolgingsplan.json

```diff
@@ -0,0 +1,23 @@
+{
+  "datoSendt": "19. september 2023",
+  "documentComponents": [
+    {
+      "type": "HEADER_H1",
+      "texts": ["Forespørsel om oppfølgingsplan"]
+    },
+    {
+      "type": "PARAGRAPH",
+      "texts": [
+        "Gjelder Artig Trane, f.nr. 12345678945"
+      ]
+    },
+    {
+      "type": "PARAGRAPH",
+      "texts": ["Nav har bedt nærmeste leder Grønn Bamse om å sende inn en oppfølgingsplan."]
```

Comment:
Vet ikke om vi bør dobbeltsjekke med noen i fag om hva som sendes her 🤔 Hvis dette eksempelet er representativt for hva vi skal sende 😊

Reply:
Ja, dette er et utkast og mest for å illustrere (det er dette som kommer opp som eksempel). Det faktiske innholdet i prod vil komme fra isoppfolgingsplan.